docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1591 @@
5ALtp8GJBr9d2o9Ah+vRT/fkAu3lyt55f/cwmZ/ddYQqY7tbakCiiggAIKKKDAigvkxI2cvfu98v+La4RwOwi4+opbWX0FFFBAAQUUUKA3Au+t9fSdV7udvX45JtAQbr051VZEAQUUUEABBRTYIJAzgXNdwPrlXkO4+epQQAEFFFBAAQV6KLAr8MFGwy8bgd8CDOHWwxNulRRQQAEFFFBgdQUM4ba6596aK6CAAgoooMAKCtwMOLTR63d+We7FEG4r+IKwygoooIACCijQX4FdysLN2dirj/X7MXDr/lbbmimggAIKKKCAAqsnYAi31Tvn1lgBBRRQQAEFVljgOsBXGz1+2fuXIdwut8IuVl0BBRRQQAEFFOidwFoh3A4H9uldba2QAgoooEDvBfJylpsCCqwtsC/wWuDytV0yhNsryt9ZtcfHvbkncC1gdyAbmfl+PK2EjMvG5S/GTdD9FVBAAQUUUEABBSYXuPIaIdw+O2UIt72BtwJ/GHApuT6ZJG+fALwNuOnk1fBIBRRQQAEFFFBAgfUELg7sD5zZaKBNG8LtasBnGmk2G3zD7uexmYabAgoooIACCiigwAwF7gz8stFIy1i+BwDbTZHPQwc0KIc19tZ6LhulD56iHB6qgAIKKKCAAgooUATWCuH2TeAGUyo9t9GgrDfucozfy4GHAXcB7lRuvxTIMYD1feu3nzVlmTxcAQUUUEABBRRYWYEqhNspjcZWjr3LXrtpJ0n9cyPdqhF3GHDbEdRvBfxkjTQePcLx7qKAAgoooIACCihQE7j5kBBuO9X2m/Rmpn/ugMbbO4Ctxkh0S+DtA9I5B7jxGOm4qwIKKKCAAgoosLICiwjhlo22nw5otOWC0ZP0KuYxeWzVg1j9/wGwxcqeSSuugAIKKKCAAgqsI5CNqMcAf2o0pE4F9ptxQ+pxjTyywfazMXv+mtXJXsOfD0j3n5o7el8BBRRQQAEFFFAAFhnCLRuaRzQaaheMOOZvvXN1u0a62bDMRqGbAgoooIACCiigQBHYHjhwwFi8nGF7hzkpZWi46hJt9f9LM8xrUCzi28wwfZNSQAEFFFBAAQU6K5CzeI9uNMYybFsuoTLOJIxxAd7YyDMbgbNcu+/hA9J/9biFdH8FFFBAAQUUUKBPAldZI4TboiJpNMfp5UzgbWcInDGD/9JoBP5ohumblAIKKKCAAgoo0BmBtUK4HQnca0G1yMZZjverLv3m/5ypO+utuTZgRivZetaZmJ4CCiiggAIKKNBmgYyi0QzhluvkZUzfaUK4jVvnvRqNv2wA5hp+s97ePSCf6886E9NTQAEFFOivwMX6WzVrtgICVyiXez83ZxFlAAAN+klEQVQH7PH/27uzGFmqMg7gf1RUFmURRRRQQUUQFGR3RUFxARMXYmLQxMRgiFF8Q18ML0ZeNLkPGpdoDMYXfLzBFVzBhUUUlVUlqAiC7KAooObT7tCcW32n73TP3KqeXyWT6a6pOnXOr+5NvtRy/hPjrQi3o0cF4AMT69f6434dB7ixY928q+ot43YpCwsBAgQIEJhJQAE4E5ONeiYwjnCryZZPmejb7aMs3Xor9qqJ9ev1cc+OA1XM3KKXmruwXfZqV/hOgAABAgSmCUgRmCZjfV8Fjk/y2SSHT3Swnrurt2/rlu9dE+vX++NOHQfsKtY6NtumVV1tdh17mxq1MQECBAhsHAEF4MY510MfaV3h+vRoSpXJOLXLkpyZ5IqeDrCeA1z00tVmFcEWAgQIECAwk4BbwDMx2Wg7Cowj3K5L8p6JLN17k3wkyct7VPzVPIPtslu7YgHfu9q8bwHtaoIAAQIENoiAK4Ab5EQPdJiHjm73tkkXXx1N6HxLz8Z1Z0d/du9YN++qrjarILYQIECAAAECBAYrMI5wq/nt6nbn+OfqNYxwWwRWvXk87uv49+cW0XDTxpc6jnNIs42vBAgQIECAAIHBCFSE281NgTOOcNux56OowrWdCPryNejzLxufShvpu80aMGiSAAECBAgQGLrAQUkubAqbuop2QZIDBjS4elZxfPWvfteE1ItM6ai3fdsro6LgBvQPRFcJECBAgACBpA8Rbos8D59pCsAqAt+1wAO8u6P9Ty6wfU0RIECAAAECBNZU4OQkNzQFzTjCbec1PfLaNX5SM54qACupZFFL11XSmhvRQoAAAQIECBDotUBFpm3uKJQuTjL0TNuaWqni3yZvAz8yiqab96Qc2/GM4fZIPJl3HPYnQIAAAQIENpBATTt0dpL7mwLpr0nq5Y/JCZ6HzFITU08WgPX5yiQVYbfapezalz+q3dNX26D9CBAgQIAAAQJrLVC3KdsC5uEkm5LssdYHX+f2n5jkmo4isOLqVrvUvm1RWW8Ym8tztaL2I0CAAAECBNZMoCLczuu4dXlpkiPX7Kjbv+FKKGnf1q0CruLstiWBp7b9VEfxV89KHrb9h6kHBAgQIECAAIFHBapwOSvJHU3xck+SM7axCHq01WF9+lAz9vEVvCp+j5hhKPU85M862qi5Bt83w/42IUCAAAECBAism0BdmfpxR+FSVwL3Wbde9ONAH+9wGBeCdQv3Y0nenuS4JPWSx9tGz0lWkTjebvJ3FX+VgWwhQIAAAQIECPRCYFqE22+TnNCLHm6fTtTVukozmSzkVvP57iRv3T5DcFQCBAgQIECAwJYCXRFu9bZv3QYWU5a8cDQf4GoKv7rq97Uk+2/Jbg0BAgQIECBAYP0FXpTkoo6rW0OLcFsvubrN+8Ukt3WYTSsOv7JenXMcAgQIENgYAssy79rGOFv9GmXl23509LzaZNbtTUnq5Yea6NmydYEqng9OUhNj75qkpo+p/5MfTrJ7s+sHknyhWecrAQIECBAgQGDdBN6Y5HfNFayhR7itG94MB3pNkgcb35oz8ZQZ9rUJAQIECBAgQGChAtMi3L6b/z/jttCDbfDG3pGkIuQmbws/kOSYDe5i+AQIECBAgMA6CWyUCLd14pz5MB9sCs
```

Comment:
I think the example is above, not below. Probably should reference via a link, not by vertical direction, in case things get reordered.

```suggestion
    "See the example above in *Indexing* to use `quadtree_point_in_polygon`.\n",
```